### PR TITLE
Fix Taiko #1, fixed Mania #1, and fix CTB #1 username

### PR DESCRIPTION
**Pull request by BlueChinchompa**
Fixed Taiko # 1, fixed Mania # 1, and fixed CTB # 1 username

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/History_of_osu!/Online_rankings/en.md [hunk 1]
```diff
@@ -24,19 +24,19 @@ As of July 19, 2021, on the [osu! performance rankings](https://osu.ppy.sh/ranki
 
 *Main page: [osu!taiko](osu!taiko)*
 
-As of June 3, 2021, on the [osu!taiko performance rankings](https://osu.ppy.sh/rankings/taiko/performance) the rank #1 player is ::{ flag=JP }:: [syaron105](https://osu.ppy.sh/users/8741695/taiko).
+As of October 4, 2021, on the [osu!taiko performance rankings](https://osu.ppy.sh/rankings/taiko/performance) the rank #1 player is ::{ flag=JP }:: [shinchikuhome](https://osu.ppy.sh/users/3174184/taiko).
```
Comment (aceticke) [inline wiki/History_of_osu!/Online_rankings/en.md:27]:
```suggestion
As of October 4, 2022, on the [osu!taiko performance rankings](https://osu.ppy.sh/rankings/taiko/performance) the rank #1 player is ::{ flag=JP }:: [shinchikuhome](https://osu.ppy.sh/users/3174184/taiko).
```

**Review by aceticke — CHANGES_REQUESTED** [review]

File: wiki/History_of_osu!/Online_rankings/en.md [hunk 2]
```diff
@@ -24,19 +24,19 @@ As of July 19, 2021, on the [osu! performance rankings](https://osu.ppy.sh/ranki
...
 
-As of June 7, 2021, on the [osu!catch performance rankings](https://osu.ppy.sh/rankings/fruits/performance) the rank #1 player is ::{ flag=KR }:: [CTB Rushia1](https://osu.ppy.sh/users/4158549/fruits).
+As of June 7, 2021, on the [osu!catch performance rankings](https://osu.ppy.sh/rankings/fruits/performance) the rank #1 player is ::{ flag=KR }:: [YesMyDarknesss](https://osu.ppy.sh/users/4158549/fruits).
 
 ## ![][osu!mania] osu!mania
 
 *Main page: [osu!mania](osu!mania)*
 
-As of June 3, 2021, on the [osu!mania performance rankings](https://osu.ppy.sh/rankings/mania/performance) the rank #1 player is ::{ flag=KR }:: [Jakads](https://osu.ppy.sh/users/259972/mania).
+As of April 13, 2023, on the [osu!mania performance rankings](https://osu.ppy.sh/rankings/mania/performance) the rank #1 player is ::{ flag=KR }:: [Dressurf](https://osu.ppy.sh/users/758406/mania).
```
Comment (WindowsMeosu) [inline wiki/History_of_osu!/Online_rankings/en.md:39]: minor correction ("dressurf" doesn't start with a capital)
```suggestion
As of April 13, 2023, on the [osu!mania performance rankings](https://osu.ppy.sh/rankings/mania/performance) the rank #1 player is ::{ flag=KR }:: [dressurf](https://osu.ppy.sh/users/758406/mania).
```

**Review by WindowsMeosu — COMMENTED** [review]
can't see any other changes that need corrections other than the capital name

**Review by TicClick — APPROVED** [review]
while this is correct, I think we should have proper history instead of updating a single line for the non-standard modes

**Review by TicClick — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**